src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTaskTest.java

```diff
@@ -410,7 +411,9 @@ private SinkRecord record(String topic, boolean nullKey, boolean nullValue, long
       nullKey ? null : "key",
       schema,
       nullValue ? null : struct,
-      offset
+      offset,
+      System.currentTimeMillis(),
```

Comment:
not necessary

Reply:
Same comment as earlier
> Resolved in chat. Keeping the changes because `maybeAddTimestamp()` takes in a long for its `timestamp` parameter and is called regardless of data streams being set.